File: 30DaysCode/Day1/dataType.cpp
#include <iostream>
#include <iomanip>
#include <limits>

using namespace std;

int main() {
    int i = 4;
    double d = 4.0;
    string s = "HackerRank ";
    
    // Declare second integer, double, and String variables.
    int j;
    double k;
    string line;
    // Read and save an integer, double, and String to your variables.
    getline(cin, line);
    j = stoi(line);

    getline(cin, line);
    k = stod(line);
    getline(cin, line);
    // Print the sum of both integer variables on a new line.
    cout << i + j << endl;
    // Print the sum of the double variables on a new line.
    cout << fixed << setprecision(1) << d + k << endl;
    // Concatenate and print the String variables on a new line
    // The 's' variable above should be printed first.
    cout << s + line << endl;
    return 0;
}

File: 30DaysCode/Day1/dataType.php
<?php
// Declare second integer, double, and String variables.
$j = fgets($handle);
// Read and save an integer, double, and String to your variables.
$k = fgets($handle);
$line = fgets($handle);
// Print the sum of both integer variables on a new line.

echo $i + $j;
echo "\n";
// Print the sum of the double variables on a new line.
echo number_format((float)$d + $k, 1, '.', ''); 
echo "\n";

//fwrite(STDOUT, $d + $k);
//fwrite(STDOUT, "\n");
// Concatenate and print the String variables on a new line
// The 's' variable above should be printed first.

echo $s.$line;
?>